tests/optimal_screen/test_optimal_screens.py

```diff
@@ -18,14 +18,15 @@
 #
 
 import os
+import sys
 import unittest
 
 # Import kano-settings from this cloned repository
-import sys
 sys.path.insert(1, '../../')
 
-verbose=False
-app='sudo ../../bin/kano-settings-onboot'
+verbose = False
+script_path = os.path.dirname(os.path.realpath(__file__))
+app = 'sudo ' + os.path.join(script_path, '..', '..', 'bin', 'kano-settings-onboot')
```

Comment:
Is it possible to use absolute paths? Just in case we need testing it from elsewhere...